employee_self_service/mobile/v1/ess.py

```diff
@@ -2057,14 +2057,17 @@ def update_task(**kwargs):
         task_doc = frappe.get_doc("Task", data.get("name"))
         task_doc.update(data)
         task_doc.save()
-        if data.get("assign_to"):
```

Comment:
Removed condition because there could be possibility that user wants to remove all assignee.... In that case if we keep this condition, then this condition won't be satisfied (since length must be greater than 0)